src/libUtils/EvmJsonResponse.cpp

```diff
@@ -34,23 +34,24 @@ namespace evmproj {
  * throws std::exception or the original exception is passed up to the caller.
  * */
 
-evmproj::CallResponse& GetReturn(const Json::Value& oldJason,
+evmproj::CallResponse& GetReturn(const Json::Value& oldJson,
                                  evmproj::CallResponse& fo) {
-  nlohmann::json newJason;
+  nlohmann::json newJson;
 
```

Comment:
Thanks for catching that one, pity we have to kill of Jason, never noticed him before :-)

Reply:
It was probably a spellcheck, but a funny one that I didn't notice, its like one of those examples where you can read a sentence clearly with letters mixed up and numbers inserted, the brain corrects.